people.md

```diff
@@ -16,7 +16,6 @@ sidebar_link: true
 ##### Omics data analysis and visualization
 <div class="flex-container">
   <img src="https://placehold.it/100x100" alt="">
-  <ul style="list-style-type:none;">
     <li><p style="margin-bottom: 0px"><a href="http://webjeda.com/online-cv/">Agnieszka Gromadka, MSc</a></p></li>
```

Comment:
You should remove ```style``` attribute from the paragraph, like so:

```html
<li><p><a href="http://webjeda.com/online-cv/">Agnieszka Gromadka, MSc</a></p></li>
```
It applies to all ```<li>``` tags below